text/0033-view-hierarchy.md

```diff
@@ -20,7 +20,7 @@ Example View Hierarchies from PoC implementations:
 
 # Proposal
 
-Capture the view hierarchy in our front-end SDKs and convert it to a common JSON representation. Add it as an [attachment](https://develop.sentry.dev/sdk/envelopes/#attachment) to the envelope. The `attachment_type` is set to `"event.view_hierarchy"` and the `content_type` is set to the `"application/json"` mime-type.
+Capture the view hierarchy in our front-end SDKs and convert it to a common JSON representation. Add it as an [attachment](https://develop.sentry.dev/sdk/envelopes/#attachment) to the envelope. The filename is `view-hierarchy.json`, the `attachment_type` is set to `"event.view_hierarchy"` and the `content_type` is set to the `"application/json"` mime-type.
```

Comment:
Do we currently expect to have to support multiple view hierarchies, similar to how we may have multiple screenshots? In that case I wonder if we should call out that the filename will be something like `view-hierarchy-{i}.json`

Reply:
Good point! I don't think it's required right now as we already have a top level `windows` property inside the `view-hierarchy.json` which should cover your use-case.